### PR TITLE
Week13

### PR DESCRIPTION
**Pull request by lagom0327**
[作業網址](http://mentor-program.co/group6/lagom0327/week13/index.php)

### CODE REVIEW

File: homeworks/week13/hw3/function/isSuperAdmin.php [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?php
+  function isSuperAdmin() {
+
+    if (isset($_SESSION['permission']) && $_SESSION['permission'] === 'super admin') return true;
+    return null;
+  }
+  return isSuperAdmin();
```
Comment (aszx87410) [inline homeworks/week13/hw3/function/isSuperAdmin.php:7]: 這邊放這行的原因是...?

**Comment by aszx87410** [comment]
看起來都沒什麼問題，做得不錯 👍

**Comment by aszx87410** [comment]
https://lidemy.github.io/mentor-program-3rd-lagom0327/homeworks/week13/hw1/
https://lidemy.github.io/mentor-program-3rd-lagom0327/homeworks/week13/hw2/

File: homeworks/week13/hw3/function/isSuperAdmin.php (hunk 1 above)
Comment (lagom0327) [reply homeworks/week13/hw3/function/isSuperAdmin.php:7]:
```php
  if (include('./function/isAdmin.php')) header('Location: ./admin.php');
  else if (include('./function/isSuperAdmin.php')) header('Location: ./super_admin.php');
  else header('Location: ./index.php');
```
因為我在 `handle_login.php` 裡實驗 `include`  是不是真的可以回傳值，所以寫了那行 ~

**Pull request merged**